indico/core/db/sqlalchemy/descriptions.py

```diff
@@ -38,12 +56,24 @@ def _description(cls):
             default=''
         )
 
+    @declared_attr
+    def render_mode(cls):
+        # Only add the column if there's a choice
+        # between several alternatives
+        if len(cls.possible_render_modes) > 1:
+            return db.Column(
+                PyIntEnum(RenderMode),
+                default=cls.default_render_mode,
+                nullable=False
+            )
+        else:
+            return None
```

Comment:
No, `declared_attr` is basically a classproperty except that sqlalchemy knows to check whether it returns something like a column, relationship, etc. I'm returning a value in `inherit_location` and it works fine